app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/strategies/PullUseCaseStrategies.java

```diff
@@ -15,6 +24,19 @@ public PullUseCaseStrategies(
 
     @Override
     public void onPullComplete() {
+        IProgramRepository programRepository = new ProgramLocalDataSource();
+        IMediaRepository mediaRepository = new MediaRepository();
+        List<Media> mediaList = mediaRepository.getAll();
+        if(mediaList==null || mediaList.size()==0){
+            mPullUseCase.notifyComplete();
+            return;
+        }
+        Media media = mediaList.get(0);
+        if(media!=null && media.getProgram()!=null && !media.getProgram()
+                .equals(programRepository.getUserProgram().getId())){
+            System.out.println("The media data will be removed");
+            PopulateDB.wipeMedia(Constants.MEDIA_FOLDER);
```

Comment:
We should use MediaRepository to remove data of DataBase and FileUtils (in this moment) for remove files from local folder